src/components/molecules/AppWrapper.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from "classnames"
+import { ReactNode } from "react"
+import AppBackButton from "../atom/AppBackButton"
+import AppBurgerMenuButton from "../atom/AppBurgerMenuButton"
+
+interface AppWrapperProps {
+    link?: string | null,
+    title?: string | null,
+    titleClassName?: string,
+    showBurger?: boolean
+    children?: ReactNode
+}
+
+const AppWrapper = ({link, title, titleClassName = '', showBurger = false, children}: AppWrapperProps): JSX.Element => {
+    return (
+        <div className="bg-page text-app-black h-screen">
+            <div className="flex flex-col justify-center mx-auto w-full md:w-1/2 lg:w-1/3 px-8 pt-8">
+                {(showBurger || link || title) &&
```

Comment:
I think we could simplify this a bit more. We are checking if showBurger, link, or title are defined to print one of them, but also in lines 21 and 30 we are checking again if showBurger, or link are defined.

To understand more the context, is the title always printed? 
If you want we could have a quick call to discuss it 😉